recipe/meta.yaml

```
@@ -10,30 +10,30 @@ source:
   sha256: eb03f2ef6340c6ac9e4398a30026f05471da99381b307aafe2f56e4cd26baaef
 
 build:
-  number: 0
+  number: 1
   # pybind11 seems incompatible on win
   skip: True  # [win]
+  # numpy<2 not available for py313
+  skip: True  # [py>=313]
   script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
-  missing_dso_whitelist:
-    - $RPATH/ld64.so.1  # [s390x]
 
 requirements:
   build:
     - {{ compiler('cxx') }}
   host:
-    - pip
+    - pip ==25.0
     - pybind11 >=2.2
     - python
     - setuptools >=0.7.0
     - wheel
   run:
-    - numpy >=1.17
+    - numpy >=1.17, <2
```

```suggestion
    - numpy >=1.17,<2
```
redundant whitespace